extensions/wikia/AnalyticsEngine/js/analytics_prod.js

```diff
@@ -141,6 +141,30 @@
 		['_setCustomVar', 5, 'LoginStatus',
 			!!window.wgUserName ? 'user' : 'anon', 3] );
 
+	/*
+	 * Remove when SOC-217 ABTest is finished
+	 */
+	/**
+	 * Get unconfirmed email AbTest user type
+	 * @returns {string}
+	 */
+	function getUnconfirmedEmailUserType() {
```

Comment:
One extra space after the function name https://github.com/Wikia/guidelines/blob/master/JavaScript/CodingConventions.md#functions